spec/features/scenarios/onboarding_a_withdrawn_participant/fip_to_fip_with_different_provider_spec.rb

```diff
@@ -108,8 +115,17 @@ def when_context(scenario)
                                                             declaration_type
           end
 
-          when_developers_transfer_the_withdrawn_participant "New SIT",
-                                                             "The Participant"
+          and_lead_provider_withdraws_participant "Original Lead Provider",
+                                                  "The Participant",
+                                                  scenario.participant_email,
+                                                  scenario.participant_type
+
+          when_school_uses_the_transfer_participant_wizard "New SIT",
+                                                           "The Participant",
+                                                           scenario.participant_email,
+                                                           scenario.participant_trn,
+                                                           scenario.participant_dob,
+                                                           same_provider: true # can't change training of withdrawn participant
```

Comment:
The UI does not allow you to change the training provider of a withdrawn participant during the transfer